### PR TITLE
3-forms elena ambitska 

### PR DESCRIPTION
**Pull request by elena-ambitska**
- added filters;
- added login page;
- added success page for success login; 

<details>

image

image

</details>

### CODE REVIEW

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/success.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="styles/main.css">
+        <title>Success</title>
+    </head>
+    <body>
+        <img src="./images/unicorn.gif" alt="success unicorn image" class="success__picture">
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/GamesProject/success.html:11]: Hey, @elena-ambitska, it's so sweeeet🥰
But please add button `Return to home page` according to the [task description](https://github.com/MastersAcademy/fe-course-basic-2023/issues/137#issue-1955947368), point 3.

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/cards.html [hunk 2]
```diff
@@ -23,11 +23,52 @@
...
                 </nav>
             </header>
+            <section class="filters">
+                <div class="filters__container">
+                    <div class="dropdown">
+                        <label>
+                            <select name="genre" id="genre" class="select__options">
+                                <option value="horror">Horror</option>
+                                <option value="racing">Racing</option>
+                                <option value="sport">Sport</option>
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/GamesProject/cards.html:38]: You've lost one option - `Shooter`

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/success.html (hunk 1 above)
Comment (elena-ambitska) [reply homeworks/olena.ambitska_elena-ambitska/GamesProject/success.html]: Hey, @vladyslav-yermolin-moc :raised_hands:
Added go-back button. Something went wrong while I was reading the task(

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/cards.html (hunk 2 above)
Comment (elena-ambitska) [reply homeworks/olena.ambitska_elena-ambitska/GamesProject/cards.html:38]: @vladyslav-yermolin-moc added. Thanks.

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/success.html [hunk 3]
```diff
@@ -10,8 +10,10 @@
     <body>
         <main class="main login">
             <section class="success__page">
-                <button type="button" class="primary__button" onclick="history.back()">
-                    <img src="././images/arrow-left-min.svg" width="22px" height="22px" alt="go back icon" class="left__arrow"> Go Back
+                <button type="button" class="primary__button">
+                    <a href="login.html" class="back__link">
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/GamesProject/success.html:14]: It should be named `Return to homepage` and lead you to the home page, i.e. index.html
Please correct this in the next task

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Pull request merged**